tasks/run-matlab-tests/v0/task.json

```diff
@@ -24,6 +24,10 @@
         {
           "name": "filterTests",
           "displayName": "Filter Tests"
+        },
+        {
+          "name": "runCustomization",
+          "displayName": "Run Customization"
```

Comment:
Test Run Customization?

Reply:
Good question. I believe we use "Test Run Customization" in the runtests doc but "Run Customization" in Jenkins at the moment. Is the "Test" redundant when we're in the context of the "Run MATLAB Tests" task?

Reply:
It's not that redundant. I don't feel strongly about "Run Customization." I am just afraid "Run Customization" might be read as "Run the Customization." Also, why not "Customization" only?

Reply:
@mw-hrastega should I update "Run Customization" to "Customize Test Run"?

Reply:
I think you can change it to "Test Run Customization."

Reply:
Okay, cool. I saw "Customize Test Run" in the Jenkins documentation so wanted to confirm that.

Reply:
Oh, I can see other display names in task.json. Yes, please use "Customize Test Run" to be compatible with other names (all being verb phrases).

Reply:
Looks like this didn't get changed to "Customize Test Run" yet?

Reply:
Updated to "Customize Test Run".